### PR TITLE
Add basic test_package example

### PR DESCRIPTION
**Pull request by czoido**
Goes with: https://github.com/conan-io/docs/pull/2853

### CODE REVIEW

**Comment by uilianries** [comment]
Using `test_package` as package name collides with the `test_package` folder. I would rename it to avoid that duplication.

**Comment by czoido** [comment]
> Using `test_package` as package name collides with the `test_package` folder. I would rename it to avoid that duplication.

I changed the name from `test_package` to `testing_packages` so there's no misunderstanding

File: README.md [hunk 1]
```diff
@@ -55,6 +55,11 @@
 - Learn how to configure settings and options and how Conan models binary compatibility.
   [Docs](https://docs.conan.io/en/2.0/tutorial/creating_packages/configure_options_settings.html)
 
+#### [Testing Conan packages: the test() method](tutorial/creating_packages/testing_packages/)
+
+- LUse a *test_package* to test that the Conan package can be consumed correctly.
```
Comment (RubenRBS) [inline README.md:60]:
```suggestion
- Use a *test_package* to test that the Conan package can be consumed correctly.
```

File: tutorial/creating_packages/testing_packages/test_package/conanfile.py [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+        cmake = CMake(self)
+        cmake.configure()
+        cmake.build()
+
+    def layout(self):
+        cmake_layout(self)
+
+    def test(self):
+        if not cross_building(self):
+            cmd = os.path.join(self.cpp.build.bindirs[0], "example")
```
Comment (memsharded) [inline tutorial/creating_packages/testing_packages/test_package/conanfile.py:25]:
```suggestion
        if can_run(self):
            cmd = os.path.join(self.cpp.build.bindir, "example")
```

File: tutorial/creating_packages/testing_packages/test_package/conanfile.py [hunk 3]
```diff
@@ -0,0 +1,26 @@
+import os
+
+from conan import ConanFile
+from conan.tools.cmake import CMake, cmake_layout
+from conan.tools.build import cross_building
```
Comment (memsharded) [inline tutorial/creating_packages/testing_packages/test_package/conanfile.py:5]:
```suggestion
from conan.tools.build import can_run
```

**Review by memsharded — APPROVED** [review]

**Pull request merged**